internal/notification/config.go

```diff
@@ -13,6 +13,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	configMapName           = "ods-notification"
+	urlProperty             = "URL"
+	methodProperty          = "Method"
+	contentTypeProperty     = "ContentType"
+	requestTemplateProperty = "requestTemplate"
+	notifyOnStatusProperty  = "NotifyOnStatus"
+	enabledProperty         = "Enabled"
```

Comment:
Actually I meant that the values need to be lowercase. E.g.:
```
methodProperty = "method"
```

I think that your change to un-export the constants is good though, too!

Reply:
Sorry, totally didn’t see this (it‘s been a IDE name refactoring gone wrong). Will fix it, sorry for messing this up 😔